Thinktecture.Relay.Server/Http/ActionFilters/RelayModuleBindingFilter.cs

```diff
@@ -1,6 +1,6 @@
 using System;
 
-namespace Thinktecture.Relay.Server.Controller
+namespace Thinktecture.Relay.Server.Http.ActionFilters
```

Comment:
This should be named `Filters` instead, because the Frameworks namespace is `Filters`, too.